modules/monitoring-contents-of-the-metrics-ui.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="contents-of-the-metrics-ui_{context}"]
+= Contents of the Metrics UI
+
+This section shows and explains the contents of the Metrics UI, a Web interface to Prometheus.
+
+The *Metrics* page is located in *Monitoring* -> *Metrics* of the {product-title} web console.
+
+image::../images/metrics-screen.png[]
+
+. Actions.
+* Add query.
+* Expand all query tables.
+* Delete all queries.
+. Hide the plot.
+. The interactive plot.
+. The catalog of available metrics.
+. Add query.
+. Run queries.
+. Query forms.
+. Expand or collapse the form.
+. The query.
+. Clear query.
+. Disable query.
+. Actions for a specific query.
+* Disable query.
+* Hide all series of the query from the plot.
+* Delete query.
+. The metrics table for a query.
+. Color assigned to the graph of the metric. Clicking the square shows or hides the metric's graph.
+
+Additionally, there is a link to the old Prometheus interface next to the title of the page.
```

Comment:
I'd say: You can access the old Prometheus interface by clicking the **Prometheus UI** link at the top of the page.

Would someone know it's the _old_ interface? Or is it just an alternate interface?

Reply:
The perspective of this whole section is "what do we have in this interface", that's why for all items I just call them out and say what they are for. You suggestion is valid for a procedure module.
It's not the best word, true. We are using _old_ to imply that people should move away from that and into the new interface developed by the OCP team. "Alternate" interface doesn't have that implication. As to whether someone would know: maybe not, and that's why we'd better tell them.